README.md

```diff
@@ -173,6 +173,7 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot)_
 - [HRConvert2](https://github.com/zelon88/HRConvert2) - Drag-and-drop file conversion server with session based authentication, automatic temporary file maintenance, and logging capability. `GPL-3.0` `PHP`
 - [Huginn](https://github.com/huginn/huginn) - Allows you to build agents that monitor and act on your behalf. `MIT` `Ruby`
 - [Kibitzr](https://kibitzr.github.io) - Lightweight personal web assistant with powerful integrations. ([Source Code](https://github.com/kibitzr/kibitzr)) `MIT` `Python`
+- [Klaxon](https://newsklaxon.org) - Klaxon enables reporters and editors to monitor scores of sites on the web for newsworthy changes. ([Source Code](https://github.com/themarshallproject/klaxon)) `MIT` `Ruby`
```

Comment:
The only working installation intructions are for a proprietary PaaS (Heroku). The self-hosted installation instructions do not seem to be up to date or in a known working state (Ubuntu 14). Unless this is fixed I don't think it can be added to the list.